### PR TITLE
answer queue tests

### PR DESCRIPTION
**Pull request by rawxfish**

### CODE REVIEW

File: answers/tests.py [hunk 1]
```diff
@@ -1,3 +1,110 @@
...
+        guess2 = Answer.objects.create(puzzle=self._puzzle, text="guess2")
+
+        guess1.set_status(Answer.CORRECT)
+        self._puzzle.refresh_from_db()
+        self.assertEqual(self._puzzle.status, Puzzle.SOLVED)
+        self.assertEqual(self._puzzle.answer, guess1.text)
 
-# Create your tests here.
+        # this should not affect puzzle status if it is already solved
+        guess2.set_status(Answer.INCORRECT)
```
Comment (erwa) [inline answers/tests.py]: Good test

File: manage.py [hunk 2]
```diff
@@ -13,12 +13,15 @@ def main():
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'smallboard.settings')
 
     if 'test' in sys.argv:
-        dotenv_file = os.path.join(BASE_DIR, ".env.test")
+        test_env_files = [".env.test.user", ".env.test"]
```
Comment (erwa) [inline manage.py]: Assuming the file loaded second overrides the settings in the first, I think we want to load `.env.test.user` second? I think of `.env.test` as the defaults, and then users might want to override or add some settings.

Comment (rawxfish) [reply manage.py]: It isn't overrides, it just loads the first file from that list and then breaks out of the loop

Comment (erwa) [reply manage.py]: Oh, oops, I didn't notice that. I got you now.

**Pull request merged**